### DISCUSSION TITLE
Detail in documentation which identity and tenant is used for each command

### DISCUSSION THREAD

**Discussion by EduardRiera** [microsoft/powerplatform-build-tools#409] [Ideas]
It is not clear to me which identity a given PAC command is using.
It seems "pac admin list" and "pac org list" is always using my windows local account's tenant.
Not sure about "pac admin create-service-principal". 
"pac org who" seems to use the identity (and tenant) of the selected row when running "pac auth list".

**Comment by petrochuk** [comment]
Pac uses your windows account as last resort if it is unable to find any auth profile created with `pac auth create`

Feel free to provide suggestions on how to improve the experience

**Reply by EduardRiera** [reply]
In my case it is not the last resort. "pac auth list" has entries (non related with my windows account), but still "pac admin list" or "pac org list" only show data related to my windows account only.
Maybe I misunderstood "last resort". What does it means in the context of running "pac org list"?

**Reply by petrochuk** [reply]
I agree it might be a little confusing and we are planning to improve it. Pac Cli has couple of profile types: Admin, Dataverse and Universal. `pac admin list` for example looks for "Admin" type of auth profiles, other commands might look for Dataverse, and then there is Universal which can be used for any commands.

![image](https://github.com/microsoft/powerplatform-build-tools/assets/30735471/3e5940c1-f5ab-4e12-beb0-7fb0a77c7c88)
